### PR TITLE
Add com.github.wjaguar.mtPaint (mtpaint-flatpak)

### PR DESCRIPTION
**Pull request by scx**
**mtPaint** is a simple painting program designed for creating icons and pixel-based artwork.
https://github.com/scx/flathub/tree/com.github.wjaguar.mtPaint
https://github.com/scx/mtpaint-flatpak

Upstream participation request:
https://github.com/wjaguar/mtPaint/issues/31

See also:
http://mtpaint.sourceforge.net
https://github.com/wjaguar/mtPaint

### CODE REVIEW

**Comment by scx** [comment]
Previous pull request:
https://github.com/flathub/flathub/pull/745

It was automatically closed because I have changed the branch name due to changing the Application ID.
https://github.com/scx/flathub/commit/e75e390389746da63e19b02c58f7cc4d2d38f9dd

Before posting a comment, make sure you are familiar with this:
 * https://github.com/flathub/flathub/pull/745#discussion_r238206983

**Comment by scx** [comment]
testbot, build com.github.wjaguar.mtPaint

**Comment by scx** [comment]
testbot, build com.github.wjaguar.mtPaint

**Comment by scx** [comment]
testbot, build com.github.wjaguar.mtPaint

**Comment by scx** [comment]
testbot, build com.github.wjaguar.mtPaint

**Comment by scx** [comment]
testbot, build com.github.wjaguar.mtPaint

**Comment by scx** [comment]
testbot, build com.github.wjaguar.mtPaint

**Comment by scx** [comment]
testbot, build com.github.wjaguar.mtPaint

File: com.github.wjaguar.mtPaint.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+app-id: com.github.wjaguar.mtPaint
+runtime: org.gnome.Platform
+runtime-version: '3.30'
```
Comment (LeandroStanger) [inline com.github.wjaguar.mtPaint.yaml]: Update runtime to 3.32

**Comment by LeandroStanger** [comment]
testbot, build com.github.wjaguar.mtPaint

**Comment by LeandroStanger** [comment]
bot, build com.github.wjaguar.mtPaint

**Comment by axredneck** [comment]
> [] No upstream participation (or app is no longer developed).

Last commit was today

**Comment by hfiguiere** [comment]
current version of mtPaint can be build with gtk3

see https://github.com/wjaguar/mtPaint/commit/60be9cc851dfd263381dde326cc8f955e0454c46

**Pull request closed without merging**